docs/sources/manage-users/manage-teams/index.md

```diff
@@ -0,0 +1,89 @@
++++
+title = "Manage teams"
+aliases =["/docs/grafana/latest/manage-users/add-or-remove-user-from-team/","/docs/grafana/latest/manage-users/create-or-remove-team/"]
+weight = 300
++++
+
+# Manage teams
+
+A _team_ is a group of users assigned to an organization on a Grafana server. Each user can belong to more than one organization and more than one team. Teams are generally managed by Organization Admins, but they can also be managed by Editors if the [editors_can_admin]({{< relref "../../administration/configuration.md#editors_can_admin" >}}) server setting is set to `true`. For more information, refer to [Organization roles]({{< relref "../../permissions/organization_roles.md" >}}).
+
+Teams members are assigned one of two permissions:
+- Member - Required to be a member of the team.
+- Admin - A member of the team that can also manage team membership, change team permissions, change team settings, and add or delete the team.
+
+> **Note:** You must have Organization Admin or Team Admin permissions, or Editor permissions with [editors_can_admin]({{< relref "../../administration/configuration.md#editors_can_admin" >}}) selected, in order to perform the tasks described in this page. Team Admins can only perform tasks that apply to their specific team.
+
+## View team list
+
+See the complete list of teams in your Grafana organization.
+
+{{< docs/shared "manage-users/view-team-list.md" >}}
+
+### Org Admin view
+
```

Comment:
Should there be a stem sentence? Otherwise, it is just an image under this heading.